docs/tutorial/reduction.rst

```diff
@@ -0,0 +1,982 @@
+.. meta::
```

Comment:
Don't understand why these two items are not 10 and 11
![image](https://github.com/ROCm/HIP/assets/164940351/9290df95-9e79-497b-9f58-05d558e4fb90)

However, these items in a different section are 10 and 11 - shouldn't they be 1 and 2
![image](https://github.com/ROCm/HIP/assets/164940351/6a7182cf-2a74-4e84-abb2-df47bbe28f3c)

Reply:
These should be a subsection of point 9. Sent them lower level.

Reply:
Fixed.